src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java

```diff
@@ -16,25 +17,20 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * Concurrency strategy which specifies an explicit core pool size. This is needed
```

Comment:
Have you check if this comment still holds? If it does, your change won't do any good.

Reply:
yeah, the thing is, HystrixServlet now provides diverging core thread size and max thread size, look in `AsyncWrapperServlet`, line 141

Reply:
Perfect